components/NewsroomArticle.js

```diff
@@ -0,0 +1,20 @@
+import articlesData from "../config/articles.json"
+
+export default function NewsroomArticle() {
+    return (
+        <div className="flex flex-col gap-3 overflow-y-auto mx-auto max-h-120" style={{width:'fit-content'}}>
+            {
+                articlesData.map((article, index) => (
+                    <a className="block px-4 py-6 rounded-md shadow border-2 border-gray-100 hover:border-gray-400 w-96 lg:mx-2 text-left" href={article.url} key={index}>
```

Comment:
Hey @mcturco, I want help from you. Can you please give me the properties that are to be needed in the hover effect of the articles shown in `Latest News` section? You haven't defined them in the Figma design file.

Reply:
You can use that "shadow" behaviour as we have in `Why AsyncAPI?` section

![image](https://user-images.githubusercontent.com/20404945/169373789-ddee51b0-21cd-4e66-a538-e61ea2295292.png)

It's the same behaviour as we have also for blog post card.